### ISSUE TITLE
Why variable instead of literal?

### ISSUE THREAD

**Issue by j4m-can** [NucciTheBoss/hot-potato-charm#18]
https://github.com/NucciTheBoss/charming-exercises/blob/20e2c30e57d8fb3d41dac342b6542f2f6d3b7b55/hot-potato/src/charm.py#L80-L81

Why did you choose to use a variable/identifier instead of the literals/strings instead?

**Comment by NucciTheBoss** [comment]
Answer doubles for NucciTheBoss/hot-potato-charm#19 as well. This is mostly a stylistic choice. I used a variable because I wanted to have ensured consistency when calling the keys in the storage bucket within various methods. 

This way if I update the key value (e.g. change `"passes"` to `"total_passes"`), I do not need to change every reference to that key in the charm. If I need to change the variable name, I can refactor every reference using my IDE. If I just used a string literal, I would have to work about catching literals such as "total_passes" or "passes_complete" when refactoring the "passes" literal to something else.
